static/js/components/LearnerSearch.js

```diff
@@ -126,6 +129,14 @@ export default class LearnerSearch extends SearchkitComponent {
   countryNameTranslations: Object = makeCountryNameTranslations();
   degreeTranslations: Object = makeDegreeTranslations();
 
+  constructor(props: Object) {
+    super(props);
+    this.WrappedLearnerResult = wrapWithProps(
+      {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
```

Comment:
It would be nice to investigate what the performance penalty was, but feel free to leave as is for this PR